tools/SetupFlow/DevHome.SetupFlow.Common/DevDriveFormatter/DevDriveFormatter.cs

```diff
@@ -16,6 +16,11 @@ namespace DevHome.SetupFlow.Common.DevDriveFormatter;
 /// </summary>
 public class DevDriveFormatter
 {
+    /// <summary>
+    /// Default allocation unit size for Dev Drives created via a virtual hard disk.

```

Comment:
Should a comment be added to ask not to keep this as uint?

Reply:
might be a type in your comment but I assume you meant adding a comment to keep it as a uint right?

Reply:
Yes.  Thank you for interpreting my message.